### PR TITLE
Update progress in real time

### PR DESCRIPTION
**Pull request by mas8074**

### CODE REVIEW

File: src/file_input/MIDI_file_class.py [hunk 1]
```diff
@@ -160,8 +163,19 @@ def parse_midi_file(self, file_name):
 
             if msg.type == 'set_tempo':
                 tempo = msg.tempo
-                
+        #print(f'The last message is: {track_messages[-1]}')
+        self.end_time = track_messages[-1].timestamp
+        print(f'end time: {self.end_time}')
```
Comment (cha2251) [inline src/file_input/MIDI_file_class.py:168]: leftover debug

File: src/file_input/file_input.py [hunk 2]
```diff
@@ -35,7 +37,9 @@ def copy_file_to_queue(self):
                 with self.accessLock:
                     if self.fileObject is None:
                         MIDIFileObject.hand_to_play = self.hand_to_play
-                        self.fileObject = MIDIFileObject(self.filename)                        
+                        self.fileObject = MIDIFileObject(self.filename)         
+                        print(self.fileObject.get_end_time())
```
Comment (cha2251) [inline src/file_input/file_input.py:41]: leftover debug

**Review by cha2251 — APPROVED** [review]
looks good! a few leftover print statements

File: src/file_input/MIDI_file_class.py (hunk 1 above)
Comment (mas8074) [inline src/file_input/MIDI_file_class.py:168]: ``
```suggestion
```

File: src/file_input/file_input.py (hunk 2 above)
Comment (mas8074) [inline src/file_input/file_input.py:41]:
```suggestion
```

**Pull request merged**